app/assets/javascripts/submit_form_ajax.js

```diff
@@ -10,11 +10,12 @@ function sendFormSubmissionAjax(dataObj, submitTo, responseEl = "") {
   }
   $.ajax({
     url: url,
-    data: dataObj,
-    success: (event, success) => {
-      if (responseEl !== "") {
-        $(responseEl).trigger('ajax:success', event);
-      }
+    data: dataObj
+  })
+  .done((event) =>{
+    if (responseEl !== "") {
+      $(responseEl).trigger('ajax:success', event);
     }
-  });
-}
\ No newline at end of file
+    if (callback) callback(event);
```

Comment:
```suggestion
    callback && callback(event);
```

Reply:
Changed it to this `  if (callback) return  callback(event);` since CodeClimate was expecting a return statement with a callback function.

Reply:
` return callback && callback(event);`? But no worries we can use anything

Reply:
Does the && convert ` callback` and `callback(event)` to a boolean?

Reply:
I now understand it...turns out, we don't even need the return...
![Screenshot 2021-02-02 at 20 07 44](https://user-images.githubusercontent.com/7622875/106635845-59bb2680-6592-11eb-8687-ebe65f19d7bb.png)